sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsRestClient.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.AI.TextAnalytics.Models;
+using Azure.Core;
+
+namespace Azure.AI.TextAnalytics
+{
+    internal partial class TextAnalyticsRestClient
+    {
+        private HttpMessage CreateHealthStatusNextPageRequest(string apiversion, string nextLink, bool? showStats)
+        {
+            var message = _pipeline.CreateMessage();
+            var request = message.Request;
+            request.Method = RequestMethod.Get;
+            var uri = new RawRequestUriBuilder();
+            uri.AppendRaw(endpoint, false);
+            uri.AppendRaw("/text/analytics/", false);
+            uri.AppendRaw(apiversion, false);
+            uri.AppendRawNextLink(nextLink, false);
+            if (showStats != null)
+            {
+                uri.AppendQuery("showStats", showStats.Value, true);
+            }
+            request.Uri = uri;
+            request.Headers.Add("Accept", "application/json, text/json");
+            return message;
+        }
+
+        /// <summary> Get details of the healthcare prediction job specified by the jobId. </summary>
+        /// <param name="apiversion"> The specific api version to use. </param>
+        /// <param name="nextLink"> The URL to the next page of results. </param>
+        /// <param name="showStats"> (Optional) if set to true, response will contain request and document level statistics. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        public async Task<Response<HealthcareJobState>> HealthStatusNextPageAsync(string apiversion, string nextLink, bool? showStats = null, CancellationToken cancellationToken = default)
```

Comment:
Metrics Advisor had to do some swagger transforms to get their NextPage* methods to generate properly - you may want to check to see if what they did applies here.